docker-e2e-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
```

Comment:
We should not introduce a new Docker Compose YAML file. Instead, we should add the missing piece -- the shared volume -- to the existing one. The existing one is _broken_ anyway because of that missing piece.

Reply:
@adaykin Let's delete this file.